CRM/Report/Form.php

```diff
@@ -1996,14 +1996,14 @@ public function whereClause(&$field, $op, $value, $min, $max) {
    * @param string $from
    * @param string $to
    * @param string $type
-   * @param string $fromTime
-   * @param string $toTime
+   * @param int $fromTime
```

Comment:
I'm concerned of the impact of changing fromTime/toTime to an int. What if the desired fromTime is 000100 (12:01 am) or any other value in the first hour of the day? I suspect that value will be interpreted as 100 and impact the datetime construction downstream.